src/components/auth/ResetPasswordForm.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import FormInput from "../common/FormInput";
+import { useNavigate } from "react-router-dom";
+import { resetPassword } from '../../util/DataBaseRequests';
+
+const ResetPasswordForm = () => {
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [message, setMessage] = useState('');
+  const navigate = useNavigate();
+  const [token, setToken] = useState('');
+
+  useEffect(() => {
+    const urlToken = new URLSearchParams(window.location.search).get('token');
+    console.log('Extracted token from URL:', urlToken);
+    if (urlToken) {
+      setToken(urlToken);
+    } else {
+      console.error('No token found in URL');
```

Comment:
This error should be communicated to the user. Not necessarily that there is no token since they would have no idea what that is but that there was a processing error and what steps they should take.

Reply:
Good point, thank you